README.md

```diff
@@ -196,6 +196,8 @@ Default mappings:
 
 `let g:JaveComplete_AutoStartServer = 0` - Disable automatic startup of server
 
+`let g:JavaComplete_ResultSort = 1` - Sort results alphabetically.
```

Comment:
Can you, please, rename variable to something like `JavaComplete_CompletionResultSort`? To let user know what it sorts exactly.